Sources/SnapshotTesting/SnapshotTesting.swift

```diff
@@ -48,7 +48,7 @@ public func assertSnapshot<S: Snapshot>(
   }()
 
   let testName: String = {
-    let testIdentifier = "\(snapshotDirectoryUrl):\(function)"
+    let testIdentifier = "\(snapshotDirectoryUrl.absoluteString):\(function)"
```

Comment:
linux's string convertible on `URL` prints the memory address.........

Reply:
:(